src/css/controls/flags/controls-hidden.less

```diff
@@ -14,8 +14,22 @@
         bottom: 0.5em;
     }
 
-    .jw-nextup-container {
-        bottom: 0;
+    &:not(.jw-flag-touch.jw-breakpoint-4):not(.jw-flag-touch.jw-breakpoint-5):not(.jw-flag-touch.jw-breakpoint-6):not(.jw-flag-touch.jw-breakpoint-7) {
+        .jw-nextup-container {
+            transform: translateY((@mobile-touch-target * 1.5));
+        }
+    }
+
+    &.jw-flag-touch.jw-state-playing {
```

Comment:
This is to accommodate the styling in touch.less: https://github.com/jwplayer/jwplayer/blob/master/src/css/controls/flags/touch.less#L1-L11

Also noticed that on mobile the nextup container did not shift down when a video was paused and the controls hid/showed. The movement only occurred when the video was playing and the controls hid/showed.

If we wanted to/were okay with the nextup container always moving when the control's visibility changed regardless of the state, this could be simplified to remove the `jw-state-playing` selector and the chain of `:not` above. Open to other thoughts.